src/applications/gi/sass/partials/_gi-profile-page.scss

```diff
@@ -175,6 +175,16 @@
       margin-top: 22.5px;
       margin-bottom: 7.5px;
     }
+    ul {
+      list-style-type: none !important;
```

Comment:
Does this need the `!important` declaration?

Reply:
Probably not in this case since it's scoped within `.your-estimated-benefits` class and I doubt there will be any other unordered lists being added in here. I'll remove it.